daml-lf/spec/daml-lf-1.rst

```diff
@@ -2005,17 +2005,17 @@ types that satisfies the following rules::
     n <ₜ ⟨ f₁ : τ₁, …, fₘ : τₘ ⟩
 
     fᵢ comes lexicographically before gᵢ
-  ——————————————————————————————————————————————————— TypeOrderStructFieldName
-    ⟨ f₁ : τ₁, …, fₘ : τₘ ⟩ <ₜ
-      ⟨ f₁ : σ₁, …, fᵢ₋1 : σᵢ₋₁, gᵢ : σᵢ, …, gₙ : σₙ ⟩
+  ——————————————————————————————————————————————————— TypeOrderStructFieldNameDiff
+    ⟨  f₁ : τ₁, …, fᵢ₋₁ : τᵢ₋₁, fᵢ : τᵢ, …, fₘ : τₘ ⟩ <ₜ
+        ⟨  f₁ : σ₁, …, fᵢ₋₁ : σᵢ₋₁, gᵢ : σᵢ, …, gₘ : σₘ⟩
 
-  ——————————————————————————————————————————————————— TypeOrderStructFieldNumber
-    ⟨ f₁ : τ₁, …, fₘ : τₘ ⟩ <ₜ
-      ⟨ f₁ : τ₁, …, fₘ : τₘ, fₘ₊₁ : τₘ₊₁ ⟩
+  ——————————————————————————————————————————————————— TypeOrderStructFieldNameShorter
+    ⟨  f₁ : τ₁, …, fₘ : τₘ ⟩ <ₜ
+         ⟨ f₁ : σ₁, …, fₘ : σₘ, fₘ₊₁ : σₘ₊₁, …, fₘ : σₘ⟩⟩
```

Comment:
The second struct contains the field `f_m` twice. I assume that second one should be `f_n`? However, this rule would be unnecessary if we follow my suggestion above.

Reply:
fixed in other PR.